apps/pushy/include/pushy_wm.hrl

```diff
@@ -35,7 +35,6 @@
                        pushy_job :: tuple(),
                        incarnation_id :: binary(),
                        requestor :: binary(),
-                       max_body_size :: binary(),
```

Comment:
is this a functionality regression @markan ? what was/is the state of that attempt?

Reply:
It was just broken code. We were keeping it in the state record of the web request, but never updating and instead fetching it separately from config. So might as well just strip that out.